charts/substra-backend/Chart.yaml

```diff
@@ -29,3 +29,5 @@ dependencies:
     repository: https://charts.bitnami.com/bitnami
     version: 11.6.5
     condition: minio.enabled
+annotations:
+  artifacthub.io/prerelease: "true"
```

Comment:
Important to remove it before merging the PR then. Might worth adding a comment in capital letter...

Reply:
good point!